http/action/src/main/java/io/knotx/databridge/http/action/HttpActionOptions.java

```diff
@@ -94,6 +95,44 @@ public HttpActionOptions setRequestTimeoutMs(long requestTimeoutMs) {
     return this;
   }
 
+  public String getResponseType() {
+    return responseType;
+  }
+
+  /**
+   * Configures the expected type of response the endpoint will return. Currently supported:
+   *<ul>
+   *   <li>
+   *     {@code json}
+   *     <p>
+   *       The response is appended to `_response` key of action's payload as parsed {@link JsonObject JsonObject}/{@link io.vertx.core.json.JsonArray JsonArray}.
+   *     </p>
+   *   </li>
+   *   <li>
+   *     {@code raw}
+   *     <p>
+   *       The response is appended to `_response` key of action's payload as a {@link JsonObject JsonObject} in the following format:
+   *       <pre>
+   *         {
+   *           "body": "<i>raw response body of the endpoint</i>",
```

Comment:
Can you please check the other solution:
```
webClientOptions { }
endpointOptions { }
responseOptions {
  payloadKey = "_response"
}
```
Then if the content type is `application/json` then we parse it as JSON and put within the `payloadKey` key. Otherwise, a response body is String value.

Reply:
Then we can easily configure predicates in `responseOptions`.

Reply:
Payload values:
```
_response {
  JSON body
  _statusCode = 200
  _headers {
    content-type = application/json
  }
}
```